### PR TITLE
docs: update comprehend doc

### PR DESCRIPTION
**Pull request by DenovVasil**
[issue](https://github.com/camunda/connectors/issues/3551)

**Description**
 I just removed redundant parts of the documentation and changed the name of the fields

**When should this change go live?**
This is part of a scheduled alpha or minor. (apply alpha or minor label)

**PR Checklist**
 My changes are for the next minor and are in /docs directory (aka /next/).

### CODE REVIEW

**Comment by akeller** [comment]
@DenovVasil, could you please use our PR template? This helps us scope and prioritize reviews. Your linked issue is completely empty. Given that we have no context for this change, @christinaausley would only be able to provide a grammatical review.

Until this is updated, I will place the `hold` label and block this from merging.

**Comment by DenovVasil** [comment]
> @DenovVasil, could you please use our PR template? This helps us scope and prioritize reviews. Your linked issue is completely empty. Given that we have no context for this change, @christinaausley would only be able to provide a grammatical review.
> 
> Until this is updated, I will place the `hold` label and block this from merging.

@akeller I updated description.
FYI I just removed redundant parts of the documentation and changed the names of the fields. So I hope the review will be easy for you 😊

**Comment by DenovVasil** [comment]
@chillleader  Tested it, works great. Thank you very much, this update helped me a lot 🙂

File: docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md:101]: These fields are exactly as they are referenced in the UI including casing, yes? If so, you have my approval 👍

**Review by christinaausley — APPROVED** [review]

**Pull request merged**
